composer.json

```diff
@@ -31,7 +31,8 @@
         "wp-coding-standards/wpcs": "*",
         "phpcompatibility/php-compatibility": "*",
         "php-parallel-lint/php-parallel-lint": "*",
-        "10up/wp_mock": "^0.4.2"
+        "10up/wp_mock": "^0.4.2",
+        "mikey179/vfsstream": "^1.6"
```

Comment:
I think this can be `bovigo` vs `mikey179` - looks like the GH repo was renamed, but Packagist retained old

Reply:
I just had a check and packagist doesn't currently have a bovigo/vfsStream package. It's probably safest to stay with the URL the vast majority are currently using 
![Screen Shot 2020-08-24 at 1 03 30 pm](https://user-images.githubusercontent.com/334808/90999543-3e72e500-e60a-11ea-9f82-2892b6ec1226.png)

Reply:
Ah, my lack of Packagist understand, was assuming GH repo mapping.